src/components/introduction/ChangeLog/changeLog.stories.tsx

```diff
@@ -24,6 +25,12 @@ export const Welcome = () => (
   <Section bgColor='transparent'>
     <Container style={{ marginTop: 50, width: 800 }}>
       <h2 style={{ marginBottom: 30 }}>Changelog</h2>
+      <ChangeLog
+        version='0.6.0'
+        date='20. April 2021'
+        isPublished={true}
```

Comment:
also if the version is not published yet set this attribute to false ( and if you dont plan to publish it immediately after the merge )

Reply:
Yes I set it to `true` because I wanted to publish it directly afterward 😄